filters/security/filter_42_ShadowWhisperersMalwareList/configuration.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "ShadowWhisperer's Malware List",
+    "description": "Malicious Sites, PUPs, Malware, Brower Hijackers, Phishing Sites",
+    "homepage": "https://github.com/ShadowWhisperer/BlockLists/blob/master/Lists/Malware",
```

the home page of the repo

```suggestion
    "homepage": "https://github.com/ShadowWhisperer/BlockLists",
```